packages/models-library/src/models_library/api_schemas_webserver/projects_nodes.py

```diff
@@ -121,6 +121,10 @@ class Config:
         }
 
 
+class NodeOutputs(InputSchemaWithoutCameCase):
+    outputs: dict[str, Any]
```

Comment:
@pcrespov I would like to make this pydantic model more restrictive. Do you have a suggestion for that?